lib/site_template/README.txt

```diff
@@ -0,0 +1,23 @@
+===============================
+Welcome to your new Jekyll Site
+===============================
+
+This file generated by Jekyll will neither be included in the site built
+when you run `jekyll build` (or `jekyll serve`) nor will it be read by
+git if you plan to have your site set up as a git repository.
```

Comment:
I never explained it in the PR body, but there's a reason why I chose this route: :smile: 
 - This document was visioned to be a sort-of welcome-note to the user, instead of being a manual on the user's blog repo. They may choose to include a dedicated `README.md` for that purpose or convert this document to a markdown document, but that wouldn't be done by default.
 - A `README.md` on a Jekyll Site - Source repo usually contains only one line:
`"Source to my personal blog powered by Jekyll"`
 - A user needn't concern themselves with having to edit this file as a README is not so important for his Jekyll Blog Source Repo., especially, if it would be ignored by git and Jekyll. 

So I decided, its best that git and Jekyll ignore this document (`README.txt`)